sklearn/linear_model/tests/test_ridge.py

```diff
@@ -938,3 +939,32 @@ def test_dtype_match_cholesky():
     assert ridge_32.predict(X_32).dtype == X_32.dtype
     assert ridge_64.predict(X_64).dtype == X_64.dtype
     assert_almost_equal(ridge_32.coef_, ridge_64.coef_, decimal=5)
+
+
+@pytest.mark.parametrize(
+    'solver', ['svd', 'cholesky', 'lsqr', 'sparse_cg', 'sag', 'saga'])
+def test_ridge_regression_dtype_stability(solver):
+    n_samples, n_features = 6, 5
+    X = rng.randn(n_samples, n_features)
```

Comment:
Isn't a global rng used here? I would prefer a local one. It makes test more reproducible.